administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -462,6 +462,14 @@ protected function getListQuery()
 		}
 		elseif (is_array($authorId))
 		{
+			// Check to see if by_me is in the array
+			if (in_array('by_me', $authorId))
```

Comment:
in_array is one of those micro-optimisations 

```
"array_slice"
"assert"
"boolval"
"call_user_func"
"call_user_func_array"
"chr"
"count"
"defined"
"doubleval"
"floatval"
"func_get_args"
"func_num_args"
"get_called_class"
"get_class"
"gettype"
"in_array"
"intval"
"is_array"
"is_bool"
"is_double"
"is_float"
"is_int"
"is_integer"
"is_long"
"is_null"
"is_object"
"is_real"
"is_resource"
"is_string"
"ord"
"strlen"
"strval"
```

```suggestion
			if (\in_array('by_me', $authorId))
```